pkg/autoscaler/autoscaler/util.go

```diff
@@ -126,13 +136,186 @@ func defaultTAC(tac *v1alpha1.TidbClusterAutoScaler) {
 		if len(tac.Spec.TiKV.Metrics) == 0 {
 			tac.Spec.TiKV.Metrics = append(tac.Spec.TiKV.Metrics, defaultMetricSpec)
 		}
+		if tac.Spec.TiKV.ScaleInThreshold == nil {
+			tac.Spec.TiKV.ScaleInThreshold = pointer.Int32Ptr(5)
+		}
+		if tac.Spec.TiKV.ScaleOutThreshold == nil {
+			tac.Spec.TiKV.ScaleOutThreshold = pointer.Int32Ptr(3)
+		}
+		if tac.Spec.TiKV.MetricsTimeDuration == nil {
+			tac.Spec.TiKV.MetricsTimeDuration = pointer.StringPtr("3m")
+		}
+		if tac.Spec.TiKV.ScaleOutIntervalSeconds == nil {
```

Comment:
Usually scaling in interval is bigger than that of scaling out.

Reply:
updated.